components/date-picker/date-picker.jsx

```diff
@@ -103,6 +99,10 @@ const propTypes = {
 		today: PropTypes.string,
 		weekDays: PropTypes.array,
 	}),
+	/**
+	 * A [Dropdown](http://react.lightningdesignsystem.com/components/inputs/) component. The props from this `Input` component will be merged and override any default props. See [Component composition with prop spread] for more information on this methodology.
```

Comment:
I think you're missing a url here for "[Component composition with prop spread]".

Reply:
Updated.